src/com_tjvendors/admin/views/vendor/tmpl/edit.php

```diff
@@ -10,20 +10,21 @@
 
 // No direct access
```

Comment:
no bs2 / 5 layouts for these?

Reply:
All was looking fine hence not created different layout